### PR TITLE
Hide Header Navigation on Detail Pages

### PR DESCRIPTION
**Pull request by MatApple**
This hides the Page navigation tabs and the sub tabs when configured to do so.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @Poltergeist

**Comment by pierlo-upitup** [comment]
I wonder if we should have a `RouterNavigation` component that various pages can pull which would have the logic to determine wether it should render or not. I'm OK with the proposed implementation in this PR but I'd like to hear your thoughts going forward. 
We are not going to introduce that many new pages in the foreseeable future, however this is one more thing to keep in mind 😉

File: src/js/pages/SystemPage.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import React from 'react';
 /* eslint-enable no-unused-vars */
 import {Hooks} from 'PluginSDK';
 import Page from '../components/Page';
+import RouterUtil from '../utils/RouterUtil';
```
Comment (jfurrow) [inline src/js/pages/SystemPage.js]: :abcd:

**Comment by jfurrow** [comment]
Other than the :abcd: this is good to go!

**Comment by kennyt** [comment]
conflicts

**Comment by MatApple** [comment]
@pierlo-upitup I agree that we should abstract the page navigation into a separate component. Currently our page navigation works but is certainly lacking elegance. I'll create a tech-debt JIRA for us to address your comments as an enhancement.

**Comment by MatApple** [comment]
Fixed the ABCs @jfurrow! Thanks

**Comment by kennyt** [comment]
conflicts

**Comment by MatApple** [comment]
retest this please

**Comment by MatApple** [comment]
retest this please

**Comment by kennyt** [comment]
retest this please

**Comment by MatApple** [comment]
retest this please

**Comment by MatApple** [comment]
retest this please

**Comment by rcorral** [comment]
retest this please

**Comment by rcorral** [comment]
retest this please

**Comment by pierlo-upitup** [comment]
@mesosphere-ci retest this please

**Comment by MatApple** [comment]
Integration tests are passing locally - http://cl.ly/2C0Z2M2f0X0b/Image%202016-06-16%20at%2011.40.38%20AM.png

**Pull request merged**